lesson07/assignment/my-title/src/components/images.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <h1 class="display-1">{{ header }}</h1>
+    <div class="col-lg-2">
```

Comment:
On desktop your images appear small and on the side. Remember with Bootstrap columns the space goes up to 12. col-lg-2 refers to large screen sizes making the images very small (only 2/12).